x/wasm/keeper/querier_test.go

```diff
@@ -708,17 +722,22 @@ func TestQueryPinnedCodes(t *testing.T) {
 			expCodeIDs: []uint64{exampleContract2.CodeID},
 		},
 	}
-	for msg, spec := range specs {
-		t.Run(msg, func(t *testing.T) {
+	for testName, spec := range specs {
+		t.Run(testName, func(t *testing.T) {
 			got, err := q.PinnedCodes(sdk.WrapSDKContext(ctx), spec.srcQuery)
+
 			if spec.expErr != nil {
 				assert.Nil(t, got)
-				assert.NotNil(t, err)
-
+				assert.EqualError(t, spec.expErr, err.Error())
 				return
-			}
+			} 
```

Comment:
Sorry, it seems that I had add one space after } in the before comment, please check it.

Reply:
Thanks for telling me! I removed it!